Sources/MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -433,25 +433,71 @@ public class CarPlayNavigationViewController: UIViewController, NavigationMapVie
+                let darkUnuseableColor: UIColor
+
+                if #available(iOS 13.0, *) {
+                    let lightTraitCollection = UITraitCollection(userInterfaceStyle: .light)
+                    let darkTraitCollection = UITraitCollection(userInterfaceStyle: .dark)
+
+                    lightUseableColor = LaneView.appearance(for: UITraitCollection(userInterfaceIdiom: .carPlay)).primaryColor.resolvedColor(with: lightTraitCollection)
+                    lightUnuseableColor = LaneView.appearance(for: UITraitCollection(userInterfaceIdiom: .carPlay)).secondaryColor.resolvedColor(with: lightTraitCollection)
+
+                    darkUseableColor = LaneView.appearance(for: UITraitCollection(userInterfaceIdiom: .carPlay)).primaryColor.resolvedColor(with: darkTraitCollection)
+                    darkUnuseableColor = LaneView.appearance(for: UITraitCollection(userInterfaceIdiom: .carPlay)).secondaryColor.resolvedColor(with: darkTraitCollection)
+                } else {
+                    // No light/dark traits are supported
+                    lightUseableColor = LaneView.appearance().primaryColor
+                    lightUnuseableColor = LaneView.appearance().secondaryColor
+
+                    darkUseableColor = LaneView.appearance().primaryColor
+                    darkUnuseableColor = LaneView.appearance().secondaryColor
+                }
+
+                var lightLanesImage = tertiaryInstruction.lanesImage(side: visualInstruction.drivingSide, direction: visualInstruction.primaryInstruction.maneuverDirection, useableColor: lightUseableColor, unuseableColor: lightUnuseableColor, size: CGSize(width: CGFloat(tertiaryInstruction.laneComponents.count) * lanesImageMaxSize.height, height: lanesImageMaxSize.height), scale: (carPlayManager.carWindow?.screen ?? UIScreen.main).scale)
+
+                var darkLanesImage = tertiaryInstruction.lanesImage(side: visualInstruction.drivingSide, direction: visualInstruction.primaryInstruction.maneuverDirection, useableColor: darkUseableColor, unuseableColor: darkUnuseableColor, size: CGSize(width: CGFloat(tertiaryInstruction.laneComponents.count) * lanesImageMaxSize.height, height: lanesImageMaxSize.height), scale: (carPlayManager.carWindow?.screen ?? UIScreen.main).scale)
+
+                if let image = lightLanesImage, let darkImage = darkLanesImage, image.size.width > lanesImageMaxSize.width {
+                    let aspectRatio = lanesImageMaxSize.width / image.size.width
+                    let scaledSize = CGSize(width: lanesImageMaxSize.width, height: lanesImageMaxSize.height * aspectRatio)
+                    lightLanesImage = image.scaled(scaledSize)
+                    darkLanesImage = darkImage.scaled(scaledSize)
+                }
+                if let image = lightLanesImage, let darkImage = darkLanesImage {
+                    tertiaryManeuver.symbolSet = CPImageSet(lightContentImage: image, darkContentImage: darkImage)
```

Comment:
#1885 pushed the logic for combining images into an image set down into the extension on VisualInstructionBanner, which seemed to make the code a bit cleaner. Something to consider, but this does appear to work.